packaging/rpm-ostree.spec

```diff
@@ -268,5 +273,4 @@ fi
 %files devel -f files.devel
 
 %changelog
-* Thu Nov 17 2022 Colin Walters <walters@verbum.org> - 2022.15-3
-- Dummy change to satisfy rpm timestamp clamping
+%autochangelog
```

Comment:
I vaguely remember that breaking some things with copr repos. Might have to look if this works with our continuous copr builds.

Reply:
I see this comment: 
https://fedoramagazine.org/use-rpmautospec-in-fedora-linux/#comment-550158

from a year ago hopefully now it's good :crossed_fingers: